python/doc/examples/functional_modeling/functional_basis/plot_multidimensional_basis.py

```diff
@@ -0,0 +1,59 @@
+"""
+============================================================================
+Create a basis of functions with input and output dimension greater than one
+============================================================================
+"""
+# %%
+# Import the required modules
```

Comment:
This comment is unnecessary, because it is strictly necessary, and any Python user knows this. It is not specific to this particular script.

Reply:
ok

Reply:
Given that any Python script *must* import the OpenTURNS library, commenting the fact that we have to import the modules is unnecessary for any user who is not a beginner. Otherwise, any Python script may have that comment, which removes its significance.